### ISSUE TITLE
About accessibility - Lack of an element of identification (title) of the iframe

### ISSUE THREAD

**Issue by felipeduardo** [collective/sc.social.like#9]
The Like button facebook uses an iframe, but the iframe does not have the "title" attribute, which is not recommended by the manuals of W3C - WCAG 2.0.

http://www.w3.org/TR/WCAG20-TECHS/H64.html

**Comment by keul** [comment]
1. Right now the FB iframe has a title ("fb:like Facebook Social Plugin", sadly not translated)
2. However I think we have no control on this, because this iframe is generated by Facebook external code.

**Comment by hvelarde** [comment]
same as https://github.com/simplesconsultoria/sc.embedder/issues/8
